syntax = "proto3";
import "protobuf/digital_id.proto";

message DigitalIdTransaction {

    bytes digital_id = 1;
    string owner_signature = 2; // calculate signature on hash of digital_id
    // new attribute 'status' added for verification by transaction family
    Status status = 3;  //this specifies the action being taken or committed
    string certifier_signature = 4;
    string receiver_address = 6;

}